packages/circuits-sdk/test/unit/CircomCommitmentWrapper.test.ts

```diff
@@ -49,8 +88,8 @@ describe("CommitmentCircuitWrapper tests", () => {
   test("Proof Validates", async () => {
     // Arrange
     const mocks = new CircomCommitmentWrapperMocks();
-    const Commitment = new CircomCommitmentWrapper();
-
+    const Commitment = mocks.factory();
+    await Commitment.preFetch();
```

Comment:
Instead of a separate await, you can just chain this in the Act section. Commitment.preFetch().andThen(() => {return commitment.prove().....